Python/hashtable.c

```diff
@@ -109,10 +110,9 @@ round_size(size_t s)
     size_t i;
     if (s < HASHTABLE_MIN_SIZE)
```

Comment:
Micro-optimisation :-)

```suggestion
    if (s <= HASHTABLE_MIN_SIZE)
```

Reply:
This actually may make the code slower. The value passed to round_size will almost never be < HASHTABLE_MIN_SIZE, but it sometimes is equal.
The branch prediction will probably be faster if it can assume the condition is always false.